tools/migrate/migrations/20240920084000_ulid-prefixes.up.sql

```diff
@@ -0,0 +1,34 @@
+-- modify "balance_snapshots" table
+ALTER TABLE "balance_snapshots" ALTER COLUMN "owner_id" TYPE character varying(34);
+-- modify "billing_invoice_items" table
+ALTER TABLE "billing_invoice_items" ALTER COLUMN "id" TYPE character varying(34), ALTER COLUMN "invoice_id" TYPE character varying(34);
+-- modify "billing_invoices" table
+ALTER TABLE "billing_invoices" ALTER COLUMN "id" TYPE character varying(34), ALTER COLUMN "billing_profile_id" TYPE character varying(34), ALTER COLUMN "workflow_config_id" TYPE character varying(34);
```

Comment:
TODO: we need to change the schema override too (or remove in case of edge fields as it seems like ent autogenerates the types)